x-pack/test/security_solution_endpoint/config.ts

```diff
@@ -39,7 +39,7 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
       ...xpackFunctionalConfig.get('kbnTestServer'),
       serverArgs: [
         ...xpackFunctionalConfig.get('kbnTestServer.serverArgs'),
-        '--xpack.ingestManager.enabled=true',
+        '--xpack.fleet.enabled=true',
```

Comment:
Maybe not for this PR, but: Fleet is now (as of 7.9 I think) enabled by default, right? so we could actually remove this flag. If so, I can follow up on our side at some point.

Reply:
👍 yes good catch